packages/cmf/src/cmfConnect.js

```diff
@@ -169,6 +169,11 @@ export default function cmfConnect({
 					this.props,
 					{ dispatchActionCreator: this.dispatchActionCreator },
 				);
+				// remove all internal props already used by the container
```

Comment:
i think it should be a delete instead of setting the props to undefined ?
if the props is still present any function checking property existence will return true.